app/components/Layout/Header.tsx

```diff
@@ -180,6 +238,11 @@ const HeaderLayout = ({
               background-color 0.15s ease-in-out, border-color 0.15s ease-in-out,
               box-shadow 0.15s ease-in-out;
           }
+          #menu-toggle.logged-in {
+            background: #f8f9fa;
+            border-radius: 100%;
+            padding: 0.375rem 0.7rem;
```

Comment:
If you set an explicit 40px height and width on this, the circular background would no longer be squished and occupy the same width as the hamburger icon 🙂 (to complement, the icon's size can be fine-tuned to fit well within by setting its `style` prop inline instead of the `size` prop)

<img width="381" alt="Screen Shot 2021-04-12 at 2 56 50 PM" src="https://user-images.githubusercontent.com/5522075/114468167-e809e500-9b9f-11eb-92f0-295020df7c47.png">

Reply:
I can probably offer some CSS suggestions later on to help trim this down to streamline the dropdown toggle's containing box and equate it with the mobile hamburger button size.

<img width="1440" alt="Screen Shot 2021-04-15 at 10 48 14 AM" src="https://user-images.githubusercontent.com/5522075/114915199-225dc700-9dd8-11eb-90b1-6bf0cf69e21c.png">

Reply:
Bootstrap menus seem to be calculated internally to a position that just looks slightly off:

<img width="275" alt="Screenshot 2021-04-16 at 6 19 37 PM" src="https://user-images.githubusercontent.com/35253/115097658-5ae8c800-9ee0-11eb-8c03-6a7c87502434.png">

On mobile the hamburger and user icon are in a button with the same height and width. Was there something else needed there?

Reply:
I can remove padding from the desktop dropdown if you want, the hamburger icon will never be shown in the menu like this.